sql/pg_scripts/ght-restore-pg

```diff
@@ -55,30 +55,52 @@ fi
 # Convert to full path
 dumpDir="`pwd`/$dumpDir"
 
-if [ ! -e $dumpDir/schema.sql ]; then
-  echo "Cannot find $dumpDir/schema.sql to create DB schema"
+pgDir=`echo $dumpDir/pg_scipts`
+
+if [ ! -e $pgDir/schema.sql ]; then
+  echo "Cannot find $pgDir/schema.sql to create DB schema"
   exit 1
 fi
 
 # 1. Create db schema
 echo "`date` Creating the DB schema"
-cat $dumpDir/schema.sql | $psql
+cat $pgDir/schema.sql | $psql
+
+count=`ls -1 $dumpDir/*.csv | wc -l`
+if [ $count != 0 ]; then
+   for f in $dumpDir/*.csv ; do
+     gzip $f
+   done
+fi
 
 # 2. Restore CSV files with disabled FK checks
 for f in $dumpDir/*.csv.gz ; do
   table=`basename $f|cut -f1 -d'.'`
   echo "`date` Restoring table $table"
-  echo "COPY $table FROM PROGRAM 'zcat < $f';" | $psql || exit 1
+  echo "COPY $table FROM PROGRAM 'zcat < $f' WITH (FORMAT 'csv', QUOTE E'\"', ESCAPE '\', NULL '\N');" | $psql || exit 1
```

Comment:
The `copy` statement here looks rather convoluted. You could do `COPY table FROM file_path'. For example, the following line works on my server:

```sql
 COPY commit_parents FROM '/home/gousiosg/ghtorrent-dump/mysql-2018-09-01/./commit_parents.csv' WITH (FORMAT 'csv', QUOTE E'"', ESCAPE '\', NULL '\N');
```